recipes/openvino/install/install_dev.sh

```diff
@@ -0,0 +1,3 @@
+# !/usr/bin/env bash
+
+cmake --install "$SRC_DIR/openvino-build" --component core_dev
```

Comment:
What's the purpose of the requirements files shipped in $PREFIX/share/openvino? This isn't a python package. Maybe add a remove step here if those files are not needed.

Reply:
Thank you, added removal cmd